django_project/changes/tests/test_github_pull_request.py

```diff
@@ -29,11 +32,13 @@ def __init__(self, json_data, status_code):
         def json(self):
             return self.json_data
 
+    if args[0] == 'https://api.github.com/users/qgis':
+        return MockResponse({
+            'html_url': 'https://github.com/qgis',
+            'name': 'name'
+        }, 200)
```

Comment:
Explain in comment, why you need to do an if switch if the argument is like that?
Why the html_url is different too.

Reply:
ah, yes. this one used for 
https://github.com/kartoza/prj.app/blob/dbf642b9185448f582fd42d3c1f28a81336b1806/django_project/changes/tests/test_github_pull_request.py#L147

I was thinking to use mock during the test, I changed the return value and arguments so that it can be utilized by another test. haha, it ended up, i didnt use it in this PR.

Ya, comment is very important to explain my code :sweat_smile: . Thanks Rizky.